truenas/webui
# NAS-112770 / 22.02-RC.2 / bootenv-create & bootenv-rename is now bootenv-form (based on ix-forms)

To test:
- Go to system/boot 
- Add a new boot environment from top actions menu btn
- Rename a bootenv using row actions.
- Make sure both forms successfully submit and close
- Make sure validation on name field works properly (description is in the tooltip)
